putiosync/webif/transmissionrpc.py

```diff
@@ -171,7 +171,7 @@ def handle_request(self):
             logger.info("Method: %r, Arguments: %r", method, arguments)
             logger.info("%r", flask.request.headers)
             try:
-                result = self.methods[method](**arguments)
+                result = self.methods[method](arguments)
```

Comment:
`**arguments` is more idiomatic, so I'll be changing it back with a trailing `**_kwargs` added to methods to support future, unhandled keyword arguments.